src/Samples/DIPS.Xamarin.UI.Samples/Controls/FloatingActionMenu/FloatingActionMenuPage.xaml

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0"
+      encoding="utf-8"?>
+
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:dxui="http://dips.xamarin.ui.com"
+             xmlns:FloatingActionMenu="clr-namespace:DIPS.Xamarin.UI.Samples.Controls.FloatingActionMenu;assembly=DIPS.Xamarin.UI.Samples"
+             mc:Ignorable="d"
+             x:Class="DIPS.Xamarin.UI.Samples.Controls.FloatingActionMenu.FloatingActionMenuPage">
+    <ContentPage.Content>
+        <RelativeLayout>
+            <RelativeLayout.Behaviors>
```

Comment:
Do you have an example of how to make this into a reusable component to share in a big project?

Reply:
Yes, I have used it like this in an app

![image](https://user-images.githubusercontent.com/22314146/74088693-44074400-4a99-11ea-8ca7-b1ef1537dad0.png)